packages/smart-wallet/src/smartWallet.js

```diff
@@ -494,6 +551,112 @@ export const prepareSmartWallet = (baggage, shared) => {
           void helper.watchPurse(purse);
           return purse;
         },
+
+        repairUnwatchedSeats() {
+          const { state, facets } = this;
+          const { address, invitationPurse } = state;
+          const { liveOffers, liveOfferSeats } = state;
+          const { zoe, agoricNames } = shared;
+          const { invitationBrand, invitationIssuer } = shared;
+
+          if (baggage.has(REPAIRED_UNWATCHED_SEATS)) {
+            return;
+          }
+          baggage.init(REPAIRED_UNWATCHED_SEATS, true);
```

Comment:
A boolean makes it looks like it could be false. Since the value is never read, use a sentinel like 'present'.

Or confirm the value is true.